File: easy/014. Sum of squares/Squares.java
import java.util.Scanner;

class Squares {
	public static void main(String[] arguments) {
		Scanner input = new Scanner(System.in);
		int from = input.nextInt();
		int end = input.nextInt();
		int total = 0;
		for (int i = from; i <= end; i++) {
			total += Math.pow(i, 2);
		}
		System.out.print(total);
	}
}

File: easy/014. Sum of squares/squares.cpp
#include <iostream>
#include <complex>
using namespace std;

int main()
{
	int from = 0;
	int end = 0;
	cin >> from >> end;
	int total = 0;
	for (int i = from; i <= end; i++) {
		total += pow(i, 2);
	}
	cout << total;
}